app/scripts/angular-apimock.js

```diff
@@ -77,38 +99,31 @@ angular.module('apiMock', [])
 			return keys;
 		}
 
-		// Taken from Angular 1.4.x: https://github.com/angular/angular.js/blob/929ec6ba5a60e926654583033a90aebe716123c0/src/ng/http.js#L18
+		// Modified from Angular 1.4.x: https://github.com/angular/angular.js/blob/929ec6ba5a60e926654583033a90aebe716123c0/src/ng/http.js#L18
 		function serializeValue(v) {
-			if (angular.isObject(v)) {
-				return angular.isDate(v) ? v.toISOString() : angular.toJson(v);
+			if (angular.isDate(v)) {
+				return toISOString(v);
 			}
+
```

Comment:
Sorry, I was rather unclear:

The function body is now

		 function serializeValue(v) {
			if (angular.isDate(v)) {
				return toISOString(v);
			}

			return v;
		}

So right now, v is not serialized if it's an object, unless I'm missing something. Below is a full example. Ternary or if statement doesn't matter, but if there would be more corner cases later, it will be cleaner to default to a ternary in my opinion :)

		function serializeValue(v) {
			if (angular.isDate(v)) {
				return toISOString(v);
			}

			return angular.isObject(v) ? angular.toJson(v) : v;
		}